src/otelcollector/otelcol-config.yml

```diff
@@ -14,7 +14,8 @@ receivers:
     targets:
       - endpoint: http://frontendproxy:${env:ENVOY_PORT}
   redis:
```

Comment:
I am wondering if the collector should rename this receiver to `datastore` or something more generic that envelopes valkey and redis

Reply:
yeah, I'd +1 that.
There are also the .NET dependencies and instrumentation that are still called Redis something.